openaq_fastapi/openaq_fastapi/v3/models/queries.py

```diff
@@ -480,7 +469,7 @@ def where(self, geometry_field: str = "geom"):
 class BboxQuery(QueryBaseModel):
     bbox: Union[str, None] = Query(
         None,
-        regex=r"^-?\d{1,2}\.?\d{0,4},-?1?\d{1,2}\.?\d{0,4},-?\d{1,2}\.?\d{0,4},-?\d{1,2}\.?\d{0,4}$",
+        regex=r"^-?\d{1,3}\.?\d{0,4},-?\d{1,2}\.?\d{0,4},-?\d{1,3}\.?\d{0,4},-?\d{1,2}\.?\d{0,4}$",
```

Comment:
I think instead of` \d{1,3}` for the x values we want to use `1?\d{1,2}` like we were before because its more correct for the actual range [-180,180]. `\d{1,3}` would allow 999 or 284 or 365 all are not valid longitudes

Reply:
good catch. I made a more specific regex that should work

Reply:
although long i think this will hit all the cases and be the most correct. I does not validate that the bbox follows the correct order in terms of xmax y max xmin ymin but that could be handled elsewhere 

`^([+-])?(?:180(?:\.0{1,4})?|((?:|[1-9]|1[0-7])[0-9])(?:\.[0-9]{1,4})?)\,([+-])?(?:90(?:\.0{1,4})?|((?:|[1-8])[0-9])(?:\.[0-9]{1,4})?)\,([+-])?(?:180(?:\.0{1,4})?|((?:|[1-9]|1[0-7])[0-9])(?:\.[0-9]{1,4})?)\,([+-])?(?:90(?:\.0{1,4})?|((?:|[1-8])[0-9])(?:\.[0-9]{1,4})?)$`